packages/arb-validator/challenges/messagesChallenge.go

```diff
@@ -162,6 +155,8 @@ func defendMessages(
 					err = contract.OneStepProofERC721Message(ctx, startInbox, startMessages, msg)
 				case message.DeliveredContractTransaction:
 					err = contract.OneStepProofContractTransactionMessage(ctx, startInbox, startMessages, msg)
+				case message.DeliveredTransactionBatch:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [extra empty line at the end of a block](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)